crypto_test.go

```diff
@@ -0,0 +1,134 @@
+package skynet
```

Comment:
All tests should have docstrings.

If you update to use the packages we already have for hashing and encoding then probably most of these tests can be removed.  Probably just the `publicKeyFromPrivateKey` function is needed.